webpack/assets/javascripts/react_app/components/common/emptyState/emptyState.test.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import DefaultEmptyState, { EmptyStatePattern } from './index';
+import { testComponentSnapshotsWithFixtures } from '../../../common/testHelpers';
+
+const defaultEmptyStateFixtures = {
+  'icon, header, description and main action are mandatory': {
+    header: 'header1',
+    description: 'description1',
+    action: { title: 'action-title', url: 'action-url' },
+  },
+  'should render secondary actions': {
+    header: 'Printers',
+    description: 'Printers print a file from the computer',
+    action: { title: 'action-1-title', url: 'action-1-url' },
+    secondaryActions: [
+      { title: 'action-2-title', url: 'action-2-url' },
+      { title: 'action-3-title', url: 'action-3-url' },
+      { title: 'action-4-title', url: 'action-4-url' },
+    ],
+  },
+};
+
+describe('Default Empty State', () => {
+  // React.Fragment is rendered as <Unknown>:
+  // https://github.com/facebook/jest/pull/5816
+  // To fix this, I am using TestRenderer.create.
```

Comment:
BTW, I tried to mock `React.Fragment` using `jest` (mock and setMock) none was working for me.
In addition, I found `TestRenderer` much easier to follow and to test.

If someone is capable to mock `React.Fragment` please let me know. :dango: